HouseRules_Essentials/README.md

```diff
@@ -75,6 +75,24 @@ An example [Ruleset for rapid play](../docs/TestingRuleSet.json) is provided to
   },
   ```
 
+- __AbilityRandomPieceListRule__: The randomPieceList for Abilities is adjusted
+  - Some abilities (NaturesCall, SpawnCultists) have a list which is used to spawn random pieces.
+  - This rule allows the list to be replaced with a different one.
+  - Config accepts Dictionary e.g. `{ "AbilityName", BoardpieceId[], "AbilityName2", BoardpieceId[] }`  
+
+  ###### _Example JSON config for AbilityRandomPieceListRule_
+
+  ```json
+  {
+  "Rule": "AbilityRandomPieceList",
+  "Config": {
+    "NaturesCall": [
+      "ChestGoblin",
+      "Slime"
+    ]
+  },
+  ```
```

Comment:
Trying to find what is wrong here now... It is not obvious. Even the linter in vscode says it's all fine.

Reply:
Not sure if this is what GitHub is complaining about, but from what I can gather from my IDE:

The `json` hint at the start of the code block may be enforcing the contents of the block to be valid JSON, which means a closing `}` is missing.

As is, my IDE is complaining as well.  The following clears it up for me:
```
  {
    "Rule": "AbilityRandomPieceList",
    "Config": {
      "NaturesCall": [
        "ChestGoblin",
        "Slime"
      ]
    }
  }
```

Specifically, the closing `}` and no trailing commas.

Reply:
It's fine with the trailing comma... Just needs the right number of `}` chars. Good spot. Couldn't see it last night no matter how many times I looked. Thanks @orendain